client/packages/system/src/Patient/PatientView/AppBarButtons.tsx

```diff
@@ -22,11 +23,19 @@ export const AppBarButtons: FC = () => {
   ) => {
     print({ reportId: report.id, dataId: patientId, args });
   };
+  const { data: enrolmentData } =
```

Comment:
Added this here to disable `Add Encounter` button but would it be better to put it in PatientView and pass it down to the AppButtons and `EncounterSearchInput`?